src/main/java/catan/domain/transfer/output/game/GameDetails.java

```diff
@@ -38,6 +39,7 @@ public GameDetails(GameBean game, int detailsRequesterId) {
         this.maxPlayers = game.getMaxPlayers();
         this.targetVictoryPoints = game.getTargetVictoryPoints();
         this.currentMove = game.getCurrentMove();
+        this.dice = new DiceDetails(game.getDiceFirstValue(), game.getDiceSecondValue());
```

Comment:
as temporary decision, I refactored it so that "dice = null" in case if dices not thrown yet. IMHO, in future we shoud create separated JSONs for new/playing status and preparation/main stage